functions.php

```diff
@@ -18,4 +18,43 @@ function getData(object $db): array {
     $query = $db->prepare('SELECT `albumName`, `artistName`, `genre`, `purchaseDate`, `albumImage` FROM `records`;');
     $query->execute();
     return $query->fetchAll();
-}
\ No newline at end of file
+}
+
+/**
+ * Description: Takes an array of collection data and outputs each item to its own <div> container, with formatting
+ * @param array $data - Collection of data fetched from database
+ * @return string
+ */
+function outputRecords(array $data): string {
+    $result = '';
+    // Check that array has required fields
+    if (!count($data)) {
+        return 'You haven\'t added to the collection yet';
+    }
+    foreach ($data as $record) {
+        // If album cover is missing, insert placeholder image
+        $placeholderImage = './images/missingCover.png';
+        if ($record['albumImage'] == NULL || $record['albumImage'] == "") {
+            $record['albumImage'] = $placeholderImage;
+        }
+        // Output each data field
+        $albumImage = '<div><img class="albumImg" src="' . $record['albumImage'] . '" alt="Album cover for ' . $record['albumName'] . ' by ' . $record['artistName'] . '" /></div>';
+        $albumName = '<h1 class="albumName">' . $record['albumName'] . '</h1>';
+        $artistName = '<h2 class="artistName">' . $record['artistName'] . '</h2>';
+        $genre = '<div><p class="genre ' . $record['genre'] . '">' . $record['genre'] . '</p></div>';
+        //If purchaseDate is NULL do not include it in output string
+        if ($record['purchaseDate'] != NULL) {
```

Comment:
Probably need to add something in here to ensure that no albums can be submitted with a future date.

Reply:
Good point. Not too concerned about this at the moment, as no data is being submitted, but will keep it in mind for Story 2, when I am validating user input.

Reply:
Okay, makes sense; giving yourself a small Technical Debt